fp: Dispute Game Blacklist task (#183)
ethereum-optimism/superchain-ops: tasks/sep/fp-recovery/001-blacklist-game/README.md

# Dispute Game Blacklist Task

## Objective

This task executes a `blacklistDisputeGame` operation in the `OptimismPortal` on behalf of the `Guardian` role. Blacklisting the dispute game prevents any withdrawals proven to be contained within the game's proposed output root from being finalized.

The batch will be executed on chain ID `11155111`, and contains `1` transactions.

### Tx #1: Blacklist Dispute Game

Executes the `blacklistDisputeGame` call to the `OptimismPortal` proxy

**Function Signature:** `blacklistDisputeGame(address)`

> **To:** `0x16Fc5058F25648194471939df75CF27A2fdC48BC`

Author: Ack - going to merge to unblock some work, need the changes to support the latest contracts release. Will follow up.

**Value:** `0 WEI`

> **Raw Input Data:** `0x7d6be8dc000000000000000000000000deaddeaddeaddeaddeaddeaddeaddeaddeaddead`

Author: https://github.com/ethereum-optimism/superchain-ops/pull/183#discussion_r1600829844

# Deputy Guardian - Blacklist Dispute Game

## Objective

This task executes a `blacklistDisputeGame` operation in the `OptimismPortal` on behalf of the `Guardian` role. Blacklisting the dispute game prevents any withdrawals proven to be contained within the game's proposed output root from being finalized.

The batch will be executed on chain ID `11155111`, and contains `1` transactions.

## Tx #1: Blacklist Dispute Game

Executes the `blacklistDisputeGame` call to the `OptimismPortal` proxy

**Function Signature:** `blacklistDisputeGame(address,address)`

**To:** `0x4220C5deD9dC2C8a8366e684B098094790C72d3c`

**Value:** `0 WEI`

**Raw Input Data:** `0x629cdd4900000000000000000000000016fc5058f25648194471939df75cf27a2fdc48bc000000000000000000000000<blacklisted----game>`

### Inputs

**\_portal:** `0x16Fc5058F25648194471939df75CF27A2fdC48BC`

**\_game:** `<user-input>`

## Preparing the Operation

1. Locate the address of the dispute game that the `Guardian` wishes to blacklist.

1. Generate the batch with `just generate-input <address>`.

> 1. Collect signatures for the action.
> 
> 1. Execute the action.
> 
> ## Signing and execution
> 
> Please see the signing and execution instructions in [SINGLE.md](../../../../SINGLE.md).

Reviewer: ```suggestion
1. Collect signatures and execute the action according to the instructions in [SINGLE.md](../../../../SINGLE.md).
```
